### ISSUE TITLE
track-ability when in near-owner state

### ISSUE THREAD

**Issue by bdetwiler** [bdetwiler/draft-detecting-unwanted-location-trackers#11]
From Eric Rescorla <[ekr@rtfm.com](mailto:ekr@rtfm.com)>, Thu, 04 May 2023 00:05 UTC
via [unwanted-trackers-request@ietf.org](mailto:unwanted-trackers-request@ietf.org)

S 3.5.1.
>   When in near-owner state, the accessory SHALL rotate its resolvable
>   and private address every 15 minutes.  This is a privacy
>   consideration to deter tracking of the accessory by non-owners when
>   it is in physical proximity to the owner.

“Can you still track a device in this state?”

**Comment by ekr** [comment]
Yes, my question was whether the owner could still track the device using the network. I.e., do nearby devices still publish reports to the network?

With that said, I don't think the use of the term "tracking" with that limited meaning is helpful here. It's all tracking.
